lametro/events.py

```diff
@@ -12,13 +12,106 @@ class LametroEventScraper(LegistarAPIEventScraper):
     EVENTSPAGE = "https://metro.legistar.com/Calendar.aspx"
     TIMEZONE = "America/Los_Angeles"
 
+    def _pair_events(self, events):
+        paired_events = []
+        unpaired_events = []
+
+        for incoming_event in events:
+            try:
+                partner_event, = [e for e in unpaired_events
+                                  if incoming_event.is_partner(e)]
+            except ValueError:
+                unpaired_events.append(incoming_event)
+
+            else:
+                unpaired_events.remove(partner_event)
+                paired_events.append(incoming_event)
+                paired_events.append(partner_event)
+
+        return paired_events, unpaired_events
+
+    def _find_partner(self, event):
+
+        results = list(self.search('/events/', 'EventId',
+                                   event.partner_search_string))
+        if results:
+            partner, = results
+            partner = LAMetroAPIEvent(partner)
+            assert event.is_partner(partner)
+            return partner
+
+        elif event.is_spanish:
+            raise ValueError("Can't find English companion for Spanish Event {}".format(event['EventId']))
+
+        else:
+            return None
+        
+
+    def api_events(self, *args, **kwargs):
+        '''
+        For meetings, Metro provides an English audio recording and a
+        Spanish recording. Due to limitations with the InSite system,
+        multiple audio recordings can't be associated with a single
+        InSite event. So, Metro creates two InSite event entries for
+        the same actual event, one Insite event entry has the English
```

Comment:
"one with the English audio and one with the Spanish audio"

Reply:
Is this a change you require?